src/app/layout.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Metadata } from "next";
+import StyledComponentsRegistry from "src/lib/registry";
+import CustomAppInit from "./CustomAppInit";
+
+export const metadata: Metadata = {
+  manifest: "/manifest.json",
+  icons: {
+    icon: "/favicon.ico",
+  },
+  description:
+    "Innovative and open-source visualization application that transforms various data formats, such as JSON, YAML, XML, CSV and more, into interactive graphs.",
```

Comment:
```suggestion
    "Innovative and open-source visualization application that produces interactive graphs from various data formats, such as JSON, YAML, XML, CSV and more.",
```

Reply:
@TallTed Thank you for suggesting these changes but I didn't change any language in the metatags, it was as it is from the original repo.
This PR only focuses on upgrading the project from the pages router to the app router.

Will fix these in another PR maybe.